tests/queries/0_stateless/02496_format_datetime_in_joda_syntax.sql

```diff
@@ -0,0 +1,142 @@
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'K');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'KK');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'KKK');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'h');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'hh');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'hhh');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'H');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'HH');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'HHH');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'k');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'kk');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'kkk');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'm');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'mm');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'mmm');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 's');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'ss');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'sss');
+
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'G123DDD');
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'G\'\'DDD');
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'G\'aaa\'DDD');
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'G\'a\'\'aa\'DDD');
+
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'G123DDD');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'G\'\'DDD');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'G\'aaa\'DDD');
+SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'G\'a\'\'aa\'DDD');
+-- { echoOff }
+
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'x'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'w'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'S'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'z'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'zz'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'zzz'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'Z'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'Z'); -- { serverError 48 }
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'b'); -- { serverError 48 }
```

Comment:
Could not find what `b` stands for in Joda or databricks docs. Maybe remove l. 130/140?

Reply:
This is the exactly case I want to test. 

Related code: 
```
default:
                        if (isalpha(*cur_token))
                            throw Exception(ErrorCodes::NOT_IMPLEMENTED, "format is not supported for {}", String(cur_token, repetitions));
```